embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/injection/CustomerLogModule.kt

```diff
@@ -63,4 +69,23 @@ internal class CustomerLogModuleImpl(
             workerThreadModule.backgroundWorker(WorkerName.REMOTE_LOGGING)
         )
     }
+
+    override val logService: LogService by singleton {
```

Comment:
I think we might need to initialize this in the `OpenTelemetryModule` instead as it will be used for initializing other data capture sources.

Reply:
I don't think the `LogService` needs to be used by other data sources, since it deals with Embrace logs only. Other sources can use the `LogWriter` which is initialized in the `OpenTelemetryModule`. On the other hand, `LogOrchestrator` might need to be moved, although its dependency on `DeliveryService` can complicate things.